### PR TITLE
Changed exception when sending msg to queue without proper url

### PR DESCRIPTION
**Pull request by rojekp**

### CODE REVIEW

File: rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala [hunk 1]
```diff
@@ -22,5 +22,14 @@ package object sqs {
         case None    => reject(MissingQueryParamRejection(n))
       }
     }
+
+    def requiredUrlParam(n: String): Directive1[String] = {
+      p.get(n) match {
+        case Some(v) => provide(v)
+        case None    => reject(WrongURLFormatRejection(n))
```
Comment (adamw) [inline rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala:29]: hm well if the parameter is absent, then the format isn't wrong, it's missing, no? :)

Comment (rojekp) [reply rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala:29]: Yes, changed back to `MissingQueryParamRejection` :)

Comment (rojekp) [reply rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala:29]: Yes, resulting exception is still the same.

Comment (adamw) [reply rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala:29]: I mean, the method `requiredUrlParam` gives a `WrongURLFormatRejection` which is not very logical

Comment (rojekp) [reply rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala]: Modified solution, added new directive.

File: rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/QueueDirectives.scala [hunk 2]
```diff
@@ -76,4 +77,18 @@ trait QueueDirectives {
       body(queueData)
     }
   }
+
+  private def checkOnlyQueueNameInUri(): Directive1[String] = {
+    extractUri.flatMap {
+      uri => {
+        val onlyQueueNameInUriPath = Option(uri.path.toString().tail)
```
Comment (adamw) [inline rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/QueueDirectives.scala:84]: doesn't this always return `Some(the uri without the first character)`?

Comment (rojekp) [reply rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/QueueDirectives.scala]: Thanks for advice, now I'm checking if there is only one segment in uri.

**Pull request merged**